cmd/res/devices/sample-devices.yaml

```diff
@@ -0,0 +1,44 @@
+# Pre-define Devices
+DeviceList:
```

Comment:
Use camelCase as discussed on other PRs
Note the map keys like `REST` are exempt from this and remain as they are